### ISSUE TITLE
i3.MessageError: Expected one of these tokens: 'px', <number>

### ISSUE THREAD

**Issue by Flurrywinde** [justahuman1/i3-grid#3]
I got `i3.MessageError: Expected one of these tokens: 'px', <number>` for both `csize` and `reset`. I suspect it has to do with my i3 version being old? `i3 version 4.14.1 (2017-09-24)` I'm pretty sure this is the version in the Ubuntu 18.04 repository... Ok, I just upgraded my i3 using the instructions here: `https://i3wm.org/docs/repositories.html`, so hopefully `csize` and `reset` will work after my next reboot. I'll still submit this issue, though, in case other Ubuntu people run into this problem.

```
$ python3 -m i3grid csize --perc 50
Traceback (most recent call last):
  File "/home/kanon/anaconda3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kanon/anaconda3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/kanon/.local/lib/python3.7/site-packages/i3grid/__main__.py", line 58, in <module>
    manager.run(cmd=action)
  File "/home/kanon/.local/lib/python3.7/site-packages/i3grid/grid.py", line 757, in run
    return self.com_map[cmd](**kwargs)
  File "/home/kanon/.local/lib/python3.7/site-packages/i3grid/grid.py", line 615, in custom_resize
    return Utils.dispatch_i3msg_com("custom", data=f"{cp}ppt")
  File "/home/kanon/.local/lib/python3.7/site-packages/i3grid/grid.py", line 222, in dispatch_i3msg_com
    return dispatcher[command](data)
  File "/home/kanon/.local/lib/python3.7/site-packages/i3grid/grid.py", line 217, in <lambda>
    lambda *d: i3.resize("set", d[0], d[0])
  File "/home/kanon/.local/lib/python3.7/site-packages/i3.py", line 412, in function
    raise response
i3.MessageError: Expected one of these tokens: 'px', <number>
$ python3 -m i3grid reset
Traceback (most recent call last):
  File "/home/kanon/anaconda3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/kanon/anaconda3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/kanon/.local/lib/python3.7/site-packages/i3grid/__main__.py", line 58, in <module>
    manager.run(cmd=action)
  File "/home/kanon/.local/lib/python3.7/site-packages/i3grid/grid.py", line 757, in run
    return self.com_map[cmd](**kwargs)
  File "/home/kanon/.local/lib/python3.7/site-packages/i3grid/grid.py", line 626, in reset_win
    return Utils.dispatch_i3msg_com(command="reset")
  File "/home/kanon/.local/lib/python3.7/site-packag
es/i3grid/grid.py", line 226, in dispatch_i3msg_com
    return dispatcher[command](w, h)
  File "/home/kanon/.local/lib/python3.7/site-packages/i3grid/grid.py", line 213, in <lambda>
    i3.resize("set", "75ppt", "75ppt")
  File "/home/kanon/.local/lib/python3.7/site-packages/i3.py", line 412, in function
    raise response
i3.MessageError: Expected one of these tokens: 'px', <number>
```

**Comment by justahuman1** [comment]
I'm not sure what version the percentage option was released (The change log on GitHub only notes the newest updates). But this does seem to be an update issue. Let me know if a restart fixes it for you.

**Comment by Flurrywinde** [comment]
Yup, it fixed it (with both the i3-grid-0.2.3b2 yaml version and b3 non-yaml).

**Comment by justahuman1** [comment]
Since I am not sure at exactly which version of i3 introduced this feature, I will add this to the README. For future users, **please update i3 if you are facing this issue**. 

If update is not possible due to your system, I am considering converting the percentage to a number (simple multiplication of screen size * percentage) if more issues about this occur.
